src/service/impl/stop-places/index.ts

```diff
@@ -99,6 +104,26 @@ export default (): IStopPlacesService => {
         return Result.err(new APIError(error));
       }
     },
+    async getStopPlaceParent(query, headers) {
+      const result = await journeyPlannerClient(headers).query<
+        GetStopPlaceParentQuery,
+        GetStopPlaceParentQueryVariables
+      >({
+        query: GetStopPlaceParentDocument,
+        variables: {
+          id: query.fromStopPlaceId,
+        },
+      });
+      if (result.errors) {
+        return Result.err(new APIError(result.errors));
+      }
+      const parentStopPlaceId = result.data.stopPlace?.parent?.id;
+      if (parentStopPlaceId) {
+        return Result.ok(parentStopPlaceId);
+      }
+
+      return Result.ok(query.fromStopPlaceId);
```

Comment:
The entur graph ql returns this when it can't find the stop place 

```
{
  "data": {
    "stopPlace": null
  }
}
```

then if the result is null, should we return error?

Reply:
pushed the change, could you check again?

Reply:
Looks good 👍